tools/scripts/jpegli_tools_test.sh

```diff
@@ -185,28 +185,6 @@ djpeg_test() {
   LD_LIBRARY_PATH="${build_dir}/lib/jpegli:${LD_LIBRARY_PATH:-}" \
     djpeg -outfile "${outfn}" -rgb "${jpgfn}"
   verify_ssimulacra2 "${infn}" "${outfn}" "${minscore}"
-
-  # Test -crop flag.
-  for geometry in 256x256+128+128 256x127+128+117; do
-    local outfn="$(mktemp -p "${tmpdir}").pnm"
-    LD_LIBRARY_PATH="${build_dir}/lib/jpegli:${LD_LIBRARY_PATH:-}" \
-      djpeg -outfile "${outfn}" -crop "${geometry}" "${jpgfn}"
-    local outfn2="$(mktemp -p "${tmpdir}").pnm"
-    convert "${infn}" -crop "${geometry}" "${outfn2}"
-    verify_ssimulacra2 "${outfn2}" "${outfn}" "${minscore}"
-  done
-
-  # Test output scaling.
-  for scale in 1/4 3/8 1/2 5/8 9/8; do
-    local scalepct="$(python3 -c "print(100.0*${scale})")%"
-    local geometry=96x128+0+0
-    local outfn="$(mktemp -p "${tmpdir}").pnm"
-    LD_LIBRARY_PATH="${build_dir}/lib/jpegli:${LD_LIBRARY_PATH:-}" \
-      djpeg -outfile "${outfn}" -scale "${scale}" -crop "${geometry}" "${jpgfn}"
-    local outfn2="$(mktemp -p "${tmpdir}").pnm"
-    convert "${infn}" -scale "${scalepct}" -crop "${geometry}" "${outfn2}"
-    verify_ssimulacra2 "${outfn2}" "${outfn}" 80
-  done
```

Comment:
was this dropped by accident?

Reply:
I was running the test locally and djpeg failed because apparently these flags were removed, they don't appear in --help or in the man page any more.